user-manual/transfer/forensic.rst

```diff
@@ -4,130 +4,169 @@
-   dashboard.
+Archivematica can be configured to extract the contents from a package, which
+includes forensic image formats as well as compressed content like ZIP files.
 
-2. Add metadata from the imaging process if desired by clicking on the report
-   button.
+However, extracting the contents of a forensic disk image can cause major
+performance issues, since disk images often contain many thousands of system
+files that need to be subsequently processed in Archivematica. This can result
+in failures. We recommend not extracting forensic image formats. This can be set
+in the processing configuration by setting the *Extract packages* job to "No".
 
-.. figure:: images/forensic-start.*
-   :align: center
-   :figwidth: 70%
-   :width: 100%
-   :alt: Click on the report button to add metadata
+If you would like to extract forensic image formats anyway, you can set the
+*Extract packages* job to "Yes". However, we recommend testing this at scale to
+ensure that it is a viable workflow for your deployment. One scalability option
+that can help to mitigate the processing load caused by extraction is turning
+off `FITS`_, which is the default characterization tool that will run on every
+extracted file. For more information, see :ref:`Preservation Action Rules
+<disable-fpr-rules>` on the Scalability page.
 
-   Click on the report button to add metadata
+Delete packages after extraction
+++++++++++++++++++++++++++++++++
 
+If you have chosen to extract files, Archivematica can be configured to delete
+the original package (the metadata and logs for the package are retained). This
+can save space in the resulting AIP. The original package can also be retained.
 
-Enter your metadata and click save before navigating back to the Transfer tab.
+To have Archivematica automatically delete forensic image formats once the files
+have been extracted, set the *Delete packages after extraction* job to "Yes". If
+you would prefer to retain the file after extraction, set the job to "No".
 
-.. figure:: images/forensic-metadata-template.*
-   :align: center
-   :figwidth: 70%
-   :width: 100%
-   :alt: Forensic disk image metadata template
+Examine contents
+++++++++++++++++
 
-   Fill in the metadata template and click save. Then navigate back to Transfer
-   to continue.
+The examine contents microservice runs `Bulk Extractor`_, a forensics tool that
+can recognize credit card numbers, social security numbers, and other patterns
+in data. The microservice creates log files that can be inspected later on.
 
-You may begin entering metadata for the next image loading to the dashboard by
-clicking on the Add Next button to the right of Start Transfer.
+Examine contents is another microservice that can result in increased processing
+time, especially if it is running on the contents of an extracted forensic disk
+image. If you are not extracting contents, running Bulk Extractor is
+questionably useful, as the disk image itself is not likely to generate any
```

Comment:
lol at "questionably" -- I think we can be more assertive here too and maybe cut the middle part of this paragraph, resulting in "If you are not extracting contents, the disk image itself is not likely to generate any results from Bulk Extractor."